framework/build.xml

```diff
@@ -247,6 +247,12 @@
     <!-- Launch all *Test.java -->
 
     <target name="test" depends="clean,jar,build-test-modules" description="run tests suite">
+        <condition property="pythonExecutable" value="${basedir}/../python/python.exe">
+            <and><os family="windows"/></and>
+        </condition>
+        <condition property="pythonExecutable" value="python">
+            <and><os family="unix"/></and>
```

Comment:
<os family="mac"/> are not handle. I think you should use 
<not>
        <os family="windows"/>
</not>